doc/source/servers/batch.md

```diff
@@ -27,6 +27,7 @@ This allows you to leverage Seldon on a large number of batch applications, incl
 We have provided a set of examples that show you how you can use the Seldon batch processing component:
 
 * [Batch Processing with Argo Workflows](../examples/argo_workflows_batch.html)
+* [Batch processing with Argo Workflows and HDFS](../examples/argo_workflows_hdfs_batch.html)
```

Comment:
This is for the previous PR example right?

Reply:
Yes, forgot to create a link in previous PR.

Reply:
Ok cool